package.json

```diff
@@ -2,12 +2,14 @@
   "name": "orientation-project-js",
   "version": "0.1.0",
   "private": true,
+  "proxy":"http://localhost:5000/",
```

Comment:
Let's remove this stuff from here as well

Reply:
Proxy removed!